src/index.js

```diff
@@ -2,9 +2,8 @@
  * Created by parikhv on 1/31/16.
  */
 import 'babel-polyfill';
-import React from 'react';
+import React from 'react'; // eslint-disable-line no-unused-vars
```

Comment:
This should not be required.  There is a way to configure eslint to not report this issue.